client/galaxy/scripts/mvc/history/history-view.js

```diff
@@ -241,6 +241,7 @@ var HistoryView = _super.extend(
 
     /** override to render pagination also */
     renderItems: function( $whereTo ){
+        console.log("RENDER ITEMS");
```

Comment:
Do we need these console entries? Additionally there already was a commented out log so now we have 2.